### PR TITLE
Fix GitHub Actions

### PR DESCRIPTION
**Pull request by jlnr**
Turns out that the syntax was broken (again) in #3. Sigh, it looked like the tests were green in the UI. Really confusing...

Let's see if this works.

### CODE REVIEW

**Comment by jlnr** [comment]
@Chadowo I have sent you an invitation to be a maintainer of this repository. Then you can also directly create branches in gosu/chingu instead of Chadowo/chingu, if you prefer so.

**Comment by jlnr** [comment]
Okay, now it fails in the macOS network spec that was also failing locally.

File: spec/chingu/network_spec.rb
Comment (jlnr) [inline spec/chingu/network_spec.rb]: @Chadowo It's likely that this makes the spec run longer than before on other platforms. I don't want to get too deep into Chingu's networking code [because it looks like Wrath is the only game that uses it](https://github.com/search?q=Chingu%3A%3AGameStates%3A%3ANetworkClient&type=code), and Wrath will remain broken even with a fixed Chingu gem because it relies on Ashton as well.

Comment (jlnr) [reply spec/chingu/network_spec.rb]: Oops, it does not use Ashton, but Texplay. So that might actually be fixable.

Comment (chadowo) [reply spec/chingu/network_spec.rb]: I see, thanks for the help!

**Comment by chadowo** [comment]
Hey there, I need to thank you first for being so helpful :D! anyhow thanks for the help with networking, its somewhat puzzling for me because I don't have much experience networking in videogames, so I appreciate it a lot.

> @Chadowo I have sent you an invitation to be a maintainer of this repository. Then you can also directly create branches in gosu/chingu instead of Chadowo/chingu, if you prefer so.

I'll accept it :+1:

**Review by chadowo — APPROVED** [review]
Everything's looking good and ready to merge if you will.

**Pull request merged**
